sopel/irc/__init__.py

```diff
@@ -108,6 +109,16 @@ def config(self):
         # TODO: Deprecate config, replaced by settings
         return self.settings
 
+    @property
+    def myinfo(self):
+        """Server's :class:`~.utils.MyInfo` information.
```

Comment:
```suggestion
        """Server/network information.
        
        :type: :class:`~.utils.MyInfo` instance
```

As shown in the similar suggestion on #1758, doing this does render correctly in Sphinx.